routes/api/members.js

```diff
@@ -8,6 +8,11 @@ const passport = require('passport');
 // Load Member model
 const Member = require('../../models/Member');
 
+// @route   GET api/profile/test
+// @desc    Tests profile route
+// @access  Public
+router.get('/test', (req, res) => res.json({ msg: 'Profile Works' }));
```

Comment:
what is this?

Reply:
was a test method when I . was figuring out something with postman, forgot to delete.